src/app/api/views/comment.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import views, status
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+
+from app.permissions import OwnProfilePermission
+
+from app.services.comment import CommentService
+from app.domain.repositories.comment import CommentRepository
+from app.api.serializers.comment import CommentSerializer
+
+
+class CommentCreateView(views.APIView):
+    permission_classes = (IsAuthenticated,)
+    service = CommentService(CommentRepository())
+    serializer_class = CommentSerializer
+
+    def post(self, request, format=None):
+        comment = self.service.post_comment(request)
+        serializer = self.serializer_class(comment)
+        return Response(serializer.data, status=status.HTTP_201_CREATED)
```

Comment:
dessa forma vc está perdendo dois recursos muito importantes:

1. validação dos dados
2. transformação de dados

Seria mais interessante vc fazer algo do tipo



```suggestion
    def post(self, request, format=None):
        serializer = self.get_serializer(data=request.data)
        serializer.is_valid(raise_exception=True)
        comment = self.service.post_comment(**serializer.validated_data)
        serializer = self.get_serializer(comment)
        return Response(serializer.data, status=status.HTTP_201_CREATED)
```